.travis.yml

```diff
@@ -7,7 +7,7 @@ cache:
 before_script:
   - chmod +x scripts/docs
 script:
-  - npm run cover
+  - npm run cover && npm run lint
```

Comment:
I would lint before test so it will fail early. Also I think you and add this as another task and do not need to combine them in the task/line.